### PR TITLE
show "Future!" for time in 10 minutes later

### PR DESCRIPTION
**Pull request by yfdyh000**
I found the #116 after a long time, I've always thought that seeing the "just now" is an illusion.

### CODE REVIEW

**Comment by yfdyh000** [comment]
Fixed a bad issue.

**Comment by yfdyh000** [comment]
If we want to ignore the time offset of users, we can raise it to 25 minutes (1500s).

**Comment by yfdyh000** [comment]
@mnoorenberghe

File: includes/bugzilla-date.js [hunk 1]
```diff
@@ -65,6 +65,9 @@ function prettydate(d) {
         weeks = 0,
         months = 0;
 
+    if (diff < -600) {
+        return 'Future!'; // Found mistakes ASAP
```
Comment (mnoorenberghe) [inline includes/bugzilla-date.js]: How about just returning the original `d` in this case instead?

Comment (yfdyh000) [inline includes/bugzilla-date.js]: I don't know, I just hope that more users find and report errors. Users can still turn off this feature ("Turn timestamps into relative dates") in settings. I think it should not fail silently.

**Comment by mnoorenberghe** [comment]
Reopen if you have a fix to address my comment.

**Pull request closed without merging**